### ISSUE TITLE
[Help]: codebook loss increase significantly during the training of vevo1.5 content-style tokenizer

### ISSUE THREAD

**Issue by josephwong14wkh** [open-mmlab/Amphion#440]
I am going to train vevo1.5 from scratch using my own dataset. During the training of content-style tokenizer, whisper reconstruction loss and chromagram loss decrease, but codebook loss increase (loss even goes up to 200 while whisper reconstruction loss = 0.57). Is this a expected behavior? Thanks for helping!

codebook loss:
![Image](https://github.com/user-attachments/assets/a8f91c54-232f-4192-bec0-c37804d5bfd7)

whisper reconstruction loss:

![Image](https://github.com/user-attachments/assets/43f0f6f4-aa66-4fd1-9f62-f481522d3b9d)

**Comment by RMSnow** [comment]
Hi @josephwong14wkh, sorry for the late reply.

I think the 0.57 for whisper loss is resonable. The 200 for codebook loss might be large, but I suspect it is because that your current training steps are too small (only 20K). Our provided checkpoint is pre-trained at 300K steps.

FYI, I have attached my training loss logs. You can see the orange curve (contentstyle_fvq16384_12.5hz) for the reference.
 
![Image](https://github.com/user-attachments/assets/83b4cd1a-e6ec-4691-83f4-855e77efc7f3)

**Comment by josephwong14wkh** [comment]
Thank you!
